### PR TITLE
Files change time

### PR DESCRIPTION
**Pull request by rickle-msft**

### CODE REVIEW

File: sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/models/CopyFileSmbInfo.java [hunk 1]
```diff
@@ -126,6 +134,28 @@ public CopyFileSmbInfo setFileLastWriteTime(String fileLastWriteTime) {
...
+    }
+
+    /**
+     * Set the fileChangeTime property: Specifies either the option to copy file last write time from a source
+     * file(source) to a target file or a time value in ISO 8601 format to set as last write time on a target file.
+     *
+     * @param fileChangeTime the fileChangeTime value to set.
+     * @return the CopyFileSmbInfo object itself.
+     */
+    public CopyFileSmbInfo setFileChangeTime(String fileChangeTime) {
```
Comment (seanmcc-msft) [inline sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/models/CopyFileSmbInfo.java]: Change Time for File Copy was cut at the last minute - I removed it in .NET here - https://github.com/Azure/azure-sdk-for-net/pull/27565.

Comment (seanmcc-msft) [inline sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/models/CopyFileSmbInfo.java:154]: Change Time for File Copy was cut at the last minute - I removed it in .NET here - https://github.com/Azure/azure-sdk-for-net/pull/27565.

Comment (rickle-msft) [reply sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/models/CopyFileSmbInfo.java]: This is a generated file, so it's coming from the swagger you gave me. Do we have a version of the swagger that doesn't have this?

Comment (seanmcc-msft) [reply sdk/storage/azure-storage-file-share/src/main/java/com/azure/storage/file/share/implementation/models/CopyFileSmbInfo.java]: I don't, can you hide this with swagger readme transforms?

**Pull request merged**